### PR TITLE
Voron 0.1 mounting brackets with/without RPi 4 for Duet 2 WiFi boards

### PR DESCRIPTION
**Pull request by DanJunior78**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Duet2Wifi mounting brackets for the  Voron 0.1

#### How was it tested? 
It´s installed on my Voron V0.1 which is still in build phase (electronics need to be connected)
#### Any background context you want to provide?
RPi is screwed directly with M2.5 screws and for the board itself it needs 4 heat insert of type M3

#### Screenshots (if appropriate)
The pic is showing the version without the pi holding version. But it fits out of the CAD data. I´ve used the back panel and the pi to be sure that it fits on the back panel and with enough space to connect the plugs.

Duet2WifiMountingBracket

#### Further notes
It fits perfect...

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please create a directory under your user for this mod, in case you have other mods in the future.  E.g.  create `printer_mods/DaGlo/Duet2Wifi_Mounting_Bracket/` and move all your files in there
- Please remove spaces from filenames
- Please add the image from this PR to the directory with your mod, and include that image in your README
- `Duet2Wifi_Mounting_Bracket.stl` is not in a printable orientation
- Please fix `printer_mods/README.md` so that your line is in alphabetical order (instead of at the end), and so the link goes to your mod directory, e.g.

```
[V0.1 Duet 2 Wifi mounting brackets](./DaGlo/Duet2Wifi_Mounting_Bracket/)
```

**Comment by DanJunior78** [comment]
Thanks for your pacience. Sure, i´ll fix all those points. I am not used to use the online GitHub version and need some time to get used to it. :)

**Comment by DanJunior78** [comment]
Should be fixed now. Hope the commit is fully uploaded

File: printer_mods/README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ like so:
 | Conqueror433 | [Voron Parts Logo](./conqueror433/Voron_Parts_Logo) | Voron Parts Logo for your back panels, 3D Print or Laser Cut | :x: | :x: | :heavy_check_mark: | :x: |
 |consibonsi | [Y-Mod SW 9mm](./consibonsi/Y-Mod_SW_9mm)| 9mm Y belt mod for SW | :x: |:x: |:x: |:heavy_check_mark:|
 | d3nny | [Filtered Power Plug with RND_165-00057](./d3nny/mains_plug_panel_for_RND_165-00057) | Filtered Power Plug for RND 165-00057 and CAT6-Plug | :x: | :x: | :heavy_check_mark: |:x:|
+|DaGlo| [V0.1 Duet 2 Wifi mounting brackets](./DaGlo/Duet2Wifi_Mounting_Bracket/) | Voron 0.1 Mounting brackets with/without RPi 4 | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:72]: Broken link, your directory is called `Voron_V0_Duet_2_WiFi_Mounting_Brackets`, but the link goes to `Duet2Wifi_Mounting_Bracket`.

File: printer_mods/DaGlo/Voron_V0_Duet_2_WiFi_Mounting_Brackets/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- 6 x M3x12
+
+- 4 x M3x5 heat inserts (should also work with the 4 mm heat inserts)
+
+## for the RPi: 
+
+- 4 x M2.5x8
+![No_RPi](Voron_V0_Duet_2_Wifi_Bracket_v14.png)
+![Main View](Duet2WifiMountingBracket.jpg)
+![No_RPi](Voron_V0_Duet_2_Wifi_Bracket_with_RPi_v14.png)
```
Comment (KiloQubit) [inline printer_mods/DaGlo/Voron_V0_Duet_2_WiFi_Mounting_Brackets/README.md:14]: The text on this last one should say `[With RPi]`  instead of `[No_Rpi]`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates!  Just a few more changes below.

**Comment by DanJunior78** [comment]
Ok, fixed both.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**